examples/TileDB.CSharp.Example/TileDB.CSharp.Example.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RootNamespace>TileDB.CSharp.Examples</RootNamespace>
-    <Version>5.0.3</Version>
+    <Version>5.0.4</Version>
```

Comment:
When we change TileDB minor version we must always change minor version of the API. This needs to be 5.1. This is what we do with TileDB-Cloud-REST, TileDB-Go and all other APIs.